react/public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
`<link rel="icon" href="./favicon.ico" />` should work here since you are in the same directory